### DISCUSSION TITLE
How exactly does this work for Radarr/Sonarr?

### DISCUSSION THREAD

**Discussion by shaunvis1** [RandomNinjaAtk/arr-scripts#150] [Q&A] [answered]
I added this for Radarr. I see the scripts show up in radarr settings. But how does it work exactly?

When does it download trailers for existing movies? Where does it put them, in the movie folder? Do they show up in Plex when you click on a movie? What about extras, will it grab "making of" documentaries and the normal extras you'd find on a Blu-Ray? And where do the extras show up in Plex?

I haven't set up Sonarr but have the same questions for that.

**Comment by RandomNinjaAtk** [comment, marked as answer]
The scripts put the data where it needs to be according to Plex’s documentation. You can review Plex’s documentation on extras for more information.

List of extras comes from: https://www.themoviedb.org/

Screenshot on the radarr/sonarr readme pages shows what it will look like… 

Edit: Wiki is open for edits, feel free to add/enhance the documentation….

**Reply by shaunvis1** [reply]
But how/when does it run? Does it constantly search for missing extras or only when you add a new title? Will it automatically grab things already in my library?

**Reply by githubusr80** [reply]
If you enabled everything, it’s all automated, which is the point of these scripts. You don’t have to do anything. Read the script logs to see what is happening…
